worker/upgradeseries/worker.go

```diff
@@ -252,13 +251,6 @@ func (w *upgradeSeriesWorker) transitionUnitsStarted(statusCount int) error {
 
 	for unit, serviceName := range unitServices {
 		svc, err := w.service.DiscoverService(serviceName)
-
```

Comment:
You added the method, but you aren't calling it, and you are no longer calling the old value.